conda_forge_tick/update_recipe/build_number.py

```diff
@@ -51,3 +58,14 @@ def update_build_number(raw_meta_yaml, new_build_number, build_patterns=None):
         raw_meta_yaml = "\n".join(lines) + "\n"
 
     return raw_meta_yaml
+
+
+def update_build_number_recipe_yaml(
```

Comment:
You are going to find that people parameterize the build number in the `context` section a lot and so this code is going to fail in those cases. I think you know this, but I wanted to be sure, so I thought i'd comment.

Reply:
Good point, nice to have it here as a todo-entry for me :)

Reply:
Done in https://github.com/regro/cf-scripts/pull/2883/commits/f8c8e66f46c5d6f0ec7f82d25c010a4649fdd904